src/components/common/Button/Button.jsx

```diff
@@ -0,0 +1,17 @@
+import Link from 'next/link'
+import styles from '@/styles/common/Button/Button.module.scss'
+
+const Button = ({ buttonContent, link }) => {
```

Comment:
Can you name it as LinkButton? because this button will be used for routing.

Reply:
Okay I'll fix it!